CMakeLists.txt

```diff
@@ -123,6 +123,8 @@ else()
 endif()
 option(CELERITY_USE_MIMALLOC "Use the mimalloc memory allocator" ${CELERITY_USE_MIMALLOC_DEFAULT})
 
+option(CELERITY_TRACY_SUPPORT "Enable Tracy profiler integration" OFF)
```

Comment:
We should have some documentation for the compile time options (like we have for env vars). Probably in `docs/installation.md`

Reply:
done.